mac_pw_pool/nightly_maintenance.sh

```diff
@@ -22,9 +22,21 @@ if [[ ! -r "$CRONLOG" ]] || [[ ! -r "$CRONSCRIPT" ]] || [[ ! -d "../.git" ]]; th
 fi
 
 relaunch_web_container() {
-  # Assume code has changed, restart container w/ latest image
+  # Assume code change or image update, restart container.
   (
+    # Prevent podman and/or sub-processes from inheriting the lock FD.
+    # This would deadlock all future runs of this script or Cron.sh
+    # Can't use `flock --close ...` here because it "hangs" in this context.
+    for fd_path in /proc/self/fd/*; do
```

Comment:
`for fd in $(/bin/ls /proc/self/fd)` saves you the (useless) intermediate variable and subsequent `basename`. But LGTM anyway.